Test/DependencyCollector/FunctionalTests/FuncTest/RDDTests.cs

```diff
@@ -1019,7 +1039,7 @@ private void ExecuteAzureSDKTests(TestWebApplication testWebApplication, int cou
         private void ValidateRddTelemetryValues(TelemetryItem<RemoteDependencyData> itemToValidate, string remoteDependencyNameExpected, string commandNameExpected, int countExpected, TimeSpan accessTimeMax, bool successFlagExpected, bool asyncFlagExpected)
         {
             DependencySourceType source = sourceExpected;
-            Assert.IsTrue(itemToValidate.Data.BaseData.Name.Contains(remoteDependencyNameExpected), "The remote dependancy name is incorrect");
+            Assert.IsTrue(itemToValidate.Data.BaseData.Name.Contains(remoteDependencyNameExpected), "The remote dependancy name is incorrect: " + remoteDependencyNameExpected + "Collected " + itemToValidate.Data.BaseData.Name);
```

Comment:
Minor: You might need a space before "Collected  " as well to better separate expected name.